### PR TITLE
Updated Unity Version, Remove ORTB DLL, Fix for Android Building

### PR DESCRIPTION
**Pull request by jonathansligh-nimbus**
1. Updated Unity Version to 2022.3.45f1
2. Removed ORTB DLL and Pasted in the C# files themselves.
3. Fixed the Android Build for the Sample App (Issues with new Unity Version).

[Monday Ticket](https://adsbynimbus.monday.com/boards/5360942780/pulses/7410528847)

### CODE REVIEW

**Review by JasonSznol — APPROVED** [review]
👍  Looks good to me but I would wait for @marcsantiago to review before merging.

File: .github/workflows/rtbworkflow.yml
Comment (marcsantiago) [inline .github/workflows/rtbworkflow.yml:1]: Maybe rename this action -> to the repo name "nimbus-unity"

**Pull request merged**
